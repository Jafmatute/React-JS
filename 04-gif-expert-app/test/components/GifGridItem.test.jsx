import {render} from "@testing-library/react";
import {GifGridItem} from "../../src/components/index.js";

describe('Pruebas en el componente <GifGridItem />',()=> {

    const title = 'Google';
    const url = 'www.google.com';

    test('debe hacer match con el snapshot', ()=> {

        const {container} = render(<GifGridItem title={title} url={url} />);

        expect(container).toMatchSnapshot();

    })
})