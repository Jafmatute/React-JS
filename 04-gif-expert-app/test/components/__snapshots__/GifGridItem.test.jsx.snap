// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en el componente <GifGridItem /> debe hacer match con el snapshot 1`] = `
<div>
  <div
    class="card"
  >
    <img
      alt="Google"
      src="http://localhost/www.google.com"
    />
    <p>
      Google
    </p>
  </div>
</div>
`;
